src/renderers/WebGLRenderer.js

```diff
@@ -71,6 +71,10 @@ THREE.WebGLRenderer = function ( parameters ) {
 	this.toneMappingExposure = 1.0;
 	this.toneMappingWhitePoint = 1.0;
 
+	this.passCamera = null;
+	this.passQuad = null;
+	this.passScene = null;
```

Comment:
I've settled on using pass* rather than post*, thanks for catching the inconsistencies, I appreicate it. :)